.github/workflows/ingest.yaml

```diff
@@ -0,0 +1,80 @@
+name: Ingest
+
+defaults:
+  run:
+    # This is the same as GitHub Action's `bash` keyword as of 20 June 2023:
+    # https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsshell
+    #
+    # Completely spelling it out here so that GitHub can't change it out from under us
+    # and we don't have to refer to the docs to know the expected behavior.
+    shell: bash --noprofile --norc -eo pipefail {0}
+
+on:
```

Comment:
I want to be explicitly clear here that this workflow does not have an automated schedule. Zika's automated workflow ([ingest-to-phylogenetic](https://github.com/nextstrain/zika/blob/573ebd7d6c6f58e26b5f02f9c7ef2473623daab2/.github/workflows/ingest-to-phylogenetic.yaml#L32)) calls on the ingest workflow and phylogenetic workflow. Once the phylogenetic workflow is set up in this repo, they can be connected the same way. 

Not asking for any changes in this PR, just wanted to make sure we're all on the same page 😄